v2_papers/2010/ITC_Demystified.json

```diff
@@ -0,0 +1,289 @@
+        "subsets": [],
+        "anonymized": true
+      },
+      {
+        "name": "KAIST-I",
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured",
+        "generation_year": 2006,
+        "covered_period": "days",
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "anonymized": true
+      },
+      {
+        "name": "KAIST-II",
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured",
+        "generation_year": 2007,
+        "covered_period": "days",
+        "details": [],
+        "subsets": [],
+        "anonymized": true
+      }
+    ]
+  },
+  "preprocessing": {
+    "performed_feature_selection": false,
+    "packet_analysis_oriented": false,
+    "flow_analysis_oriented": true,
+    "flow_aggregation_analysis_oriented": false,
+    "tools": [],
+    "normalization_type": "missing",
+    "transformations": [
+      "flow_extraction",
+      "feature_aggregation"
+    ],
+    "final_data_format": "numerical_vectors",
+    "flows": [
+      {
+        "selection": "expert_knowledge",
+        "role": "main",
+        "main_goal": "traffic_classification",
+        "active_timeout": 64,
+        "idle_timeout": 64,
+        "bidirectional": false,
+        "features": [
+          "protocolIdentifier",
+          "sourceTransportPort",
+          "destinationTransportPort",
+          "packetTotalCount",
+          "octetTotalCount",
+          "flowStartMilliseconds",
+          "flowEndMilliseconds",
+          "flowDurationMilliseconds",
+          {
+            "divide": [
+              "packetTotalCount",
+              "flowDurationMilliseconds"
+            ]
+          },
+          {
+            "divide": [
+              "octetTotalCount",
+              "flowDurationMilliseconds"
+            ]
+          },
+          {},
+          {
+            "maximum": [
+              "ipTotalLength"
+            ]
+          },
+          {
+            "minimum": [
+              "ipTotalLength"
+            ]
+          },
+          {
+            "mean": [
+              "ipTotalLength"
+            ]
+          },
+          {
+            "stdev": [
+              "ipTotalLength"
+            ]
+          },
+          {
+            "maximum": [
+              "_interPacketTimeMicroseconds"
+            ]
+          },
+          {
+            "minimum": [
+              "_interPacketTimeMicroseconds"
+            ]
+          },
+          {
+            "mean": [
+              "_interPacketTimeMicroseconds"
+            ]
+          },
+          {
+            "stdev": [
+              "_interPacketTimeMicroseconds"
+            ]
+          },
+          "tcpFinTotalCount",
+          "tcpSynTotalCount",
+          "tcpRstTotalCount",
+          "tcpPshTotalCount",
+          "tcpAckTotalCount",
+          "tcpUrgTotalCount",
+          "tcpCweTotalCount",
+          "tcpEceTotalCount"
```

Comment:
``"tcpCweTotalCount"`` (is Cwe a typo for Cwr?) and ``"tcpEceTotalCount"`` don't exist in the IANA spec. I would agree that these should be useful features though, and so they should be written as ``"_tcpEceTotalCount"`` and added to the list of valid features of the verifier (I can take care of that if necessary).